include/ginkgo/core/factorization/cholesky.hpp

```diff
@@ -88,6 +90,26 @@ class Cholesky
     /** Creates a new parameter_type to set up the factory. */
     static parameters_type build() { return {}; }
 
+    const parameters_type& get_parameters() const { return parameters_; }
```

Comment:
Please put this below the non-const version and add a doxygen `copydoc` comment.